include/dem/pw_contact_force.h

```diff
@@ -128,6 +146,9 @@ class PWContactForce
       }
   }
 
+  void
+  get_force_torque();
+
```

Comment:
Add description for the function.

Reply:
This one won't be used. It's for me, to disp the information i want.